CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ##### Enhancements
 
+* Added obfuscation for keys/passwords to ensure that they don't appear in plaintext.
```

Comment:
Very minor, maybe just mention the "Obfuscation" subspec and the `ObfuscatedKey` class so that clients can get a general sense of how they can make use of this new feature when they see these release notes.

Reply:
done!